BREAKING_CHANGES.md

```diff
@@ -4,6 +4,10 @@ NOTE: many style-related COMPONENT_IDs were renamed for consistency based
 on the CSS-in-JS restructure. All component-level theming customizations
 should be re-checked for ID naming accuracy.
 
+## @zendeskgarden/react-breadcrumbs
+
+- `<Item>` has been removed.
```

Comment:
A little explainer for how the updated breadcrumb handles children would go a long way.

Reply:
Oh yeah, that would be more helpful. How does this sound?

> The `<Item>` component has been removed. The `<Breadcrumb>` component determines the last breadcrumb item and applies styling to each item. Instead of passing `<Item>` components as children to `<Breadcrumb>`, consumers can now pass in inline elements of their choice. For example, a child of `<Breadcrumb>` can be a `<span>` or a component that renders an `<a>`.

Reply:
brilliant.